etc/pip-requires-test

```diff
@@ -6,7 +6,7 @@ pytest-xdist~=2.2.0; python_version >= '3.6'      # Used for parallel testing on
 pyflakes==2.1.1                                   # Passive checker of Python programs
```

Comment:
update to 2.3.1?

Reply:
see see https://github.com/rucio/rucio/pull/4588